From 87586ae31729f6cd228a03994c9e9213ead604be Mon Sep 17 00:00:00 2001
From: Ian Wahbe <ian@wahbe.com>
Date: Thu, 2 Mar 2023 15:34:08 +0100
Subject: [PATCH] Remove `RequiredWith` for user.expire_password_on_create

---
 okta/resource_okta_user.go | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/okta/resource_okta_user.go b/okta/resource_okta_user.go
index 263ddd34..12e977f1 100644
--- a/okta/resource_okta_user.go
+++ b/okta/resource_okta_user.go
@@ -289,11 +289,12 @@ func resourceUser() *schema.Resource {
 				Description: "User Password",
 			},
 			"expire_password_on_create": {
-				Type:         schema.TypeBool,
-				Optional:     true,
-				Default:      false,
-				Description:  "If set to `true`, the user will have to change the password at the next login. This property will be used when user is being created and works only when `password` field is set",
-				RequiredWith: []string{"password"},
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				Description: "If set to `true`, the user will have to change the password at the next login. This property will be used when user is being created and works only when `password` field is set",
+				// The bridge doesn't understand RequiredWith, so we remove this check.
+				// RequiredWith: []string{"password"},
 			},
 			"password_inline_hook": {
 				Type:             schema.TypeString,
-- 
2.39.1

