From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ian Wahbe <ian@wahbe.com>
Date: Thu, 18 Apr 2024 17:12:53 +0200
Subject: [PATCH] ensure patch


diff --git a/go.mod b/go.mod
index 7e650867..fa530c99 100644
--- a/go.mod
+++ b/go.mod
@@ -1,4 +1,4 @@
-module github.com/okta/terraform-provider-okta
+module github.com/okta/terraform-provider-okta // should be v4
 
 go 1.24.0
 
